test/fixtures/foo-bar-resolver-no-version.js

```diff
@@ -1,12 +1,14 @@
+var assert = require('assert/strict')
 var path = require('path')
 
-exports.resolveImport = function (modulePath, sourceFile, config) {
+exports.resolveImport = function (modulePath, sourceFile, config, _, options) {
   var sourceFileName = path.basename(sourceFile)
   if (sourceFileName === 'foo.js') {
     return path.join(__dirname, 'bar.jsx')
   }
   if (sourceFileName === 'exception.js') {
     throw new Error('foo-bar-resolver-v1 resolveImport test exception')
   }
-  return undefined
+  assert.ok(!_, 'the 4th argument must be undefined')
+  assert.ok(options.context.cwd, 'the `context.cwd` must be present')
```

Comment:
We are still using jest, use `expect` style here.

Reply:
Hmm, it's a resolver implementation, I don't know would it be suitable to use jest API here.

Reply:
What I mean is that we can simply `throw TypeError` here, and use `expect#toThrow` in the unit test cases.

Reply:
There is no error thrown, the assertions pass.

Reply:
It is crucial to use `expect` for a better test message and report.